### PR TITLE
[thread-cert] fix case 5.5.2 random fail

### PR DESCRIPTION
**Pull request by FiveDimensions**

### CODE REVIEW

File: tests/scripts/thread-cert/Cert_5_5_02_LeaderReboot.py [hunk 1]
```diff
@@ -89,7 +89,7 @@ def test(self):
         self.assertEqual(self.nodes[ROUTER].get_state(), 'leader')
 
         self.nodes[LEADER].start()
-        self.simulator.go(5)
+        self.simulator.go(10)
```
Comment (librasungirl) [inline tests/scripts/thread-cert/Cert_5_5_02_LeaderReboot.py:92]: Is it necessary to change the wait time?
I think `5s` might be enough for the `Leader` to attach in theory.

**Comment by librasungirl** [comment]
Is there any regression results with this PR? Could it pass stably now? @FiveDimensions

File: tests/scripts/thread-cert/Cert_5_5_02_LeaderReboot.py [hunk 2]
```diff
@@ -117,14 +117,12 @@ def verify(self, pv):
 
         # Step 4: Router_1 MUST attempt to reattach to its original partition by
         # sending MLE Parent Requests to the All-Routers multicast
-        # address (FFxx::xx) with a hop limit of 255.
-        _rpkts.filter_mle_cmd(MLE_PARENT_REQUEST).must_next().must_verify(
-            lambda p: {MODE_TLV, CHALLENGE_TLV, SCAN_MASK_TLV, VERSION_TLV} == set(p.mle.tlv.type))
+        # address (FFxx::xx) with a hop limit of 255. MUST make two separate attempts
+        for i in range(1, 2):
```
[the remark this answers is not in the document]
Comment (FiveDimensions) [reply tests/scripts/thread-cert/Cert_5_5_02_LeaderReboot.py]: need to filter 2 times

File: tests/scripts/thread-cert/Cert_5_5_02_LeaderReboot.py [hunk 3]
```diff
@@ -117,14 +117,12 @@ def verify(self, pv):
...
+            _rpkts.filter_mle_cmd(MLE_PARENT_REQUEST).must_next().must_verify(
+                lambda p: {MODE_TLV, CHALLENGE_TLV, SCAN_MASK_TLV, VERSION_TLV} == set(p.mle.tlv.type))
```
[the remark this answers is not in the document]
Comment (FiveDimensions) [reply tests/scripts/thread-cert/Cert_5_5_02_LeaderReboot.py]: added

File: tests/scripts/thread-cert/Cert_5_5_02_LeaderReboot.py (hunk 1 above)
Comment (FiveDimensions) [reply tests/scripts/thread-cert/Cert_5_5_02_LeaderReboot.py]: supposed to wait more time and see if packets not caught, change back to 5 sec

**Comment by FiveDimensions** [comment]
re-run the packet verification more than 10 times, and no related issue found

**Review by simonlingoogle — APPROVED** [review]
LGTM :+1:

**Review by librasungirl — APPROVED** [review]

**Review by jwhui — APPROVED** [review]

**Pull request merged**